rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+First, you need to find the order ID in the PDF.
+
+As you can see in xref:automation-usecase-legacysystemdataextraction.adoc#confirmationletter-image[the image of the order confirmation letter], the order ID is always listed in a table next to the entry 'Order ID'. 
+// Do we need an image of that part of the letter here?
+You can use this information to implement the workflow that contains the following steps:
+
+. Copy the content of the order confirmation to a text file.
+. Read the text file line by line to find the one containing the order ID.
+. Check if the order ID is found
+. Cut the order ID out from the text and store it in an activity parameter.
+
+image::usecase-legacysystem-extractformpdf-workflow-overview.png["The Workflow contains the steps in form of Action Steps. Some of the Action Steps are nested."]
```

Comment:
Can you use callouts (to be consistent with our style) instead of boxes?

Reply:
alt text - maybe The Activity Workflow listing transitions and nested action steps"?

Reply:
I'd rather not use callouts because I want to show where the overlaying windows belong to. (You have to double-click an element in the workflow to open them.) Diane and I decided on that in the review of the first draft a while ago. Also because of the re-org-related tasks, I don't have the time to do all the screenshots again.